components/calendar/calendar.js

```diff
@@ -430,6 +465,10 @@ class Calendar extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 			getCalendarData(true);
 			this.requestUpdate();
 		});
+
+		if (this.minValue && this.maxValue && (getDateFromISODate(this.minValue).getTime() > getDateFromISODate(this.maxValue).getTime())) {
+			console.warn('d2l-calendar component expects min-value to be before max-value');
```

Comment:
I added a warning for if max is before min - not sure if that's the best way to indicate this?

Reply:
Hmm, or you could even throw a [RangeError](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RangeError)? They'll likely experience unexpected things otherwise, no?